_shared/project/.github/workflows/ci.yml

```diff
@@ -92,9 +92,9 @@ jobs:
       {% endif %}
       {% if job in ["tests", "functests"] and cookiecutter.get("postgres") == "yes" %}
       - name: Create test database
-        run: psql -U postgres -h localhost -p {{ cookiecutter['__postgres_port'] }} -c 'CREATE DATABASE {{ cookiecutter.package_name }}_test'
+        run: psql -U postgres -h localhost -p {{ cookiecutter['__postgres_port'] }} -c 'CREATE DATABASE {{ cookiecutter.package_name }}_{{ job }}'
```

Comment:
I've fixed the cookiecutter's `tox.ini` to use different DBs for the unit tests, functests and BDD tests so that `make sure` can run them in parallel. This line just fixes `ci.yml` to create the right DB name for each job.